cloudbuild-build-and-deploy.yaml

```diff
@@ -0,0 +1,44 @@
+steps:
+  # Builds the Docker image
+  - name: docker
+    id: build_proxy_service_image
+    entrypoint: "sh"
+    args:
+      - "-c"
+      - |
+        echo "*************************************************************"
+        echo "* Building Docker image for environment...                  *"
+        echo "*************************************************************"
+        docker build -t "$_EQ_CIR_PROXY_SERVICE_IMAGE_REPO:${SHORT_SHA}" .
+
+  # Pushes the Docker image to Artifact Registry
+  - name: docker
+    id: push_proxy_service_image
+    entrypoint: "sh"
+    waitFor:
+      - build_proxy_service_image
+    args:
+      - "-c"
+      - |
+        echo "*************************************************************"
+        echo "* Pushing Docker image to Artifact Registry...              *"
+        echo "*************************************************************"
+        docker push "$_EQ_CIR_PROXY_SERVICE_IMAGE_REPO:${SHORT_SHA}"
+
+  # Deploys the Docker image to Cloud Run
+  - name: "gcr.io/google.com/cloudsdktool/cloud-sdk:alpine"
+    id: deploy_proxy_service
+    entrypoint: "sh"
+    args:
+      - "-c"
+      - |
+        echo "*************************************************************"
+        echo "* Deploying Docker image to Cloud Run...                    *"
+        echo "*************************************************************"
+        gcloud run deploy cir-proxy-service \
```

Comment:
I think this might lose all of the vars we're passing in for our Terraform deployment? E.g. `LOG_LEVEL` isn't being passed in here